cdk/lib/security-hq.ts

```diff
@@ -124,9 +125,41 @@ dpkg -i /tmp/installer.deb`,
           new GuDynamoDBWritePolicy(this, 'DynamoWrite', {
             tableName: cfnTable.tableName as string,
           }),
+          new GuAllowPolicy(this, 'SsmCommands', {
```

Comment:
Somehow I'd missed the load of custom permissions SHQ needs, but I've tested and fairly confident I've not missed anything now.